src/main/perf/LineFileDocs.java

```diff
@@ -230,14 +233,16 @@ private void open() throws IOException {
 
             // Verify facet fields now:
             for(String field : facetFields.keySet()) {
-              if (field.equals("Date") == false && field.equals("Month") == false && field.equals("DayOfYear") == false && !extraFacetFieldsList.contains(field)) {
+              if (field.equals("Date") == false && field.equals("Month") == false && field.equals("DayOfYear") == false
+                      && field.equals("RandomLabel") == false && !extraFacetFieldsList.contains(field)) {
```

Comment:
Pre-existing: can we use `== false` instead of `!` on the last check, to reduce risk of future accidental refactoring bugs as has happened to Lucene sources?  Crazy that this `if` statement mixed the two like that!  Smelly :)  Prolly my fault!